app/components/Testimonial/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import './testimonial.css'
+import useIsMobile from '../../hooks/isMobile'
+
+interface TestimonialProps {
+  className?: string
+  title: string
+  description: string
+  src: string
+  layout?: 'expanded' | 'squeezed'
+}
+
+const Testimonial: React.FC<TestimonialProps> = ({
+  className,
+  title,
+  description,
+  src,
+  layout = 'squeezed',
+}) => {
+  const isMobile = useIsMobile()
+  const finalLayout = isMobile ? 'squeezed' : layout
+  className = `testimonial ${finalLayout} ${className || ''}`
+
+  return finalLayout === 'squeezed' ? (
```

Comment:
can't this be done with css + a media query?

Reply:
Yeah, that was what I thought at first, but the layouts are just different. Like the puzzle doesn't fit. I can't articulate why, just look at the different versions and you can see what I mean.

Reply:
that's unfortunate, but happens. @melissasamworth it's all your fault!!